### PR TITLE
Set the collate on the path col.

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

**Comment by jimmidyson** [comment]
Reckon wet can get a test on for this?

**Comment by chirino** [comment]
We can't really unit test against PostgreSQL can we?

**Comment by jimmidyson** [comment]
Integration test? System test?

File: jsondb/src/main/java/com/redhat/ipaas/jsondb/impl/SqlJsonDB.java [hunk 1]
```diff
@@ -138,10 +153,6 @@ public String createKey() {
         final Handle h = dbi.open();
         try {
             String sql = "select path,value,kind from jsondb where path LIKE :like order by path";
-            if( databaseKind == DatabaseKind.PostgreSQL ) {
-                // To get lexical sorting on even the / character.
-                sql = "select path,value,kind from jsondb where path LIKE :like order by BYTEA(path)";
-            }
```
Comment (rhuss) [inline jsondb/src/main/java/com/redhat/ipaas/jsondb/impl/SqlJsonDB.java]: Sorry, overlooked that. Forget my previous comment.

**Pull request merged**